sites/public/pages/account/favorites.tsx

```diff
@@ -0,0 +1,110 @@
+import Head from "next/head"
+import {
+  PageHeader,
+  AgPagination,
+  t,
+  encodeToFrontendFilterString,
+  LoadingOverlay,
+  ListingFilterState,
+  AuthContext,
+} from "@bloom-housing/ui-components"
+import Layout from "../../layouts/application"
+import { MetaTags } from "../../src/MetaTags"
+import React, { useEffect, useState, useContext, useMemo } from "react"
+import { useRouter } from "next/router"
+import { useListingsData } from "../../lib/hooks"
+import { OrderByFieldsEnum } from "@bloom-housing/backend-core/types"
+import { getListings } from "../../lib/helpers"
+
+const FavoritedListingsPage = () => {
+  const router = useRouter()
+  const { profile } = useContext(AuthContext)
+
+  // Pagination state
+  const [currentPage, setCurrentPage] = useState<number>(1)
+
+  const filterState = useMemo(() => {
+    const filterState: ListingFilterState = {
+      favorited: profile?.preferences?.favoriteIds?.toString(),
+    }
+    return filterState
+  }, [profile])
+  const itemsPerPage = 8
+
+  const setQueryString = (page: number, filters = filterState) => {
+    void router.push(
+      `/account/favorites?page=${page}${encodeToFrontendFilterString(filters)}`,
+      undefined,
+      {
+        shallow: true,
+      }
+    )
+  }
+
+  // Checks for changes in url params.
+  useEffect(() => {
+    if (router.query.page) {
+      setCurrentPage(Number(router.query.page))
+    }
+  }, [router.query])
+
+  const { listingsData, listingsLoading, listingsError } = useListingsData(
+    currentPage,
+    itemsPerPage,
+    filterState,
+    OrderByFieldsEnum.mostRecentlyUpdated
+  )
+
+  const pageTitle = `${t("pageTitle.favorites")} - ${t("nav.siteTitle")}`
+  const metaDescription = t("pageDescription.welcome", { regionName: t("region.name") })
+  const metaImage = "" // TODO: replace with hero image
+
+  return (
+    <Layout>
+      <Head>
```

Comment:
The <Layout> component already has the head/title, metatags, pageheader / loadingoverlay